src/KubernetesRuntime/KubernetesRuntime.Autorest/custom/Disable-AzKubernetesRuntimeLoadBalancer.ps1

```diff
@@ -0,0 +1,187 @@
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code
+# is regenerated.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Disable Arc load balancer service in a connected cluster.
+.Description
+Disable Arc load balancer service in a connected cluster.
+
+.Outputs
+System.String
+.Link
+https://learn.microsoft.com/powershell/module/az.kubernetesruntime/disable-azkubernetesruntimeloadbalancer
+#>
+
+function Disable-AzKubernetesRuntimeLoadBalancer {
+    [OutputType([Microsoft.Azure.PowerShell.Cmdlets.KubernetesRuntime.Models.IServiceResource])]
+    [CmdletBinding(DefaultParameterSetName = 'CreateExpanded', PositionalBinding = $false, SupportsShouldProcess, ConfirmImpact = 'Medium')]
+    param(
+        [Parameter(Mandatory)]
+        [Alias('ResourceUri')]
+        [Microsoft.Azure.PowerShell.Cmdlets.KubernetesRuntime.Category('Path')]
+        [System.String]
+        # The resource uri of the connected cluster 
+        ${ArcConnectedClusterUri},
```

Comment:
Propse to use ArcConnectedClusteId instead of ArcConnectedClusterUri for it doesn't contain hostname
```suggestion
        ${ArcConnectedClusterId},
```